### ISSUE TITLE
How to hide and show menus using methods?

### ISSUE THREAD

**Issue by acenario** [uzysjung/UzysSlideMenu#4]
Hello! 

I am big fan of your framework, however, I have been having trouble editing it to match mine and possibly the needs of others. I am using a navigation bar item to display the menu so it naturally appears over my TableViewController, however, when I click the button again it should hide the whole menu. The reason is that the first menu prevents a user from clicking the first row in the tableview.

Currently my solution is to set the itemView.alpha = 0 in the showMainMenu method and the showIconMenu method. I have also tried setting userInteractionEnabled to false and tried making it hidden, however, I still cannot click the first row.

Any suggestions or thoughts on this issue?

Thanks,

Arjun

**Comment by uzysjung** [comment]
Do you have a screenshot? 

I don't get it. please explain more details.

try 
self.uzysSMenu.hidden = YES;
[self.uzysSmenu openIconMenu];

**Comment by acenario** [comment]
Okay I finally got it to work! Thanks for the help. However, the menu doesn't disappear via the animation because I am using the .hidden property. Is there anyway I could show the animation effect when hiding menu? Ideally I would love to be able to do [self.uzysSmenu showFullMenu] and [self.uzysSmenu hideMenu].

As you asked, I have attached images of what I am trying to accomplish. My issue was that the menu was blocking me from clicking the first row of my tableview.

![screen shot 2013-08-19 at 9 34 28 am](https://f.cloud.github.com/assets/591849/986234/8d422d32-08d4-11e3-8cbb-7e41e81bbad1.png)

![screen shot 2013-08-19 at 9 34 52 am](https://f.cloud.github.com/assets/591849/986235/919ea0fe-08d4-11e3-90e1-bc4d9d284f5a.png)
